packages/assurance/README.md

```diff
@@ -6,6 +6,11 @@
 
 `@adobe/react-native-aepassurance` is a wrapper around the iOS and Android [AEPAssurance SDK](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-experience-platform-assurance) to allow for integration with React Native applications. Functionality to start Assurance session is provided through JavaScript documented below.
 
+## Prerequisites
+
+The Adobe Experience Platform Identity for Edge Network extension has the following peer dependency, which must be installed prior to installing the identity extension:
```

Comment:
This is for Edge Identity, do you want to say Assurance?

Reply:
copy pasta.. Yes I will change it.
Thank you!